backend/core/package.json

```diff
@@ -112,7 +112,7 @@
     "testEnvironment": "node"
   },
   "engines": {
-    "node": "14",
+    "node": "12",
```

Comment:
Can we do this change as a separate PR so it's clear what came from upstream?

Reply:
I think upstream is also [version 12](https://github.com/bloom-housing/bloom/blob/master/backend/core/package.json).  But I can change this back and do a separate PR for it if you want.

If we want to be super strict about it, I can remove the style change that I added to fix the logo styling that was affected when I pulled down the upstream changes.

This is what it looks like after the merge without my additional style changes:
![image](https://user-images.githubusercontent.com/16543000/122125550-9b8c9100-cdfe-11eb-8f54-61cc48866a77.png)

I removed my style changes for now.

Reply:
Also if we want to be strict about not making any additional changes, I should probably revert my fix in CHANGELOG.md?